client/settings/deposits/index.js

```diff
@@ -0,0 +1,230 @@
+/**
+ * External dependencies
+ */
+import React, { useContext } from 'react';
+import { __ } from '@wordpress/i18n';
+import {
+	Card,
+	SelectControl,
+	ExternalLink,
+	Notice,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import WCPaySettingsContext from '../wcpay-settings-context';
+import CardBody from '../card-body';
+import {
+	useDepositScheduleInterval,
+	useDepositScheduleWeeklyAnchor,
+	useDepositScheduleMonthlyAnchor,
+	useDepositStatus,
+	useCompletedWaitingPeriod,
+} from '../../data';
+import './style.scss';
+
+const daysOfWeek = [
+	{ label: __( 'Monday', 'woocommerce-payments' ), value: 'monday' },
+	{ label: __( 'Tuesday', 'woocommerce-payments' ), value: 'tuesday' },
+	{
+		label: __( 'Wednesday', 'woocommerce-payments' ),
+		value: 'wednesday',
+	},
+	{ label: __( 'Thursday', 'woocommerce-payments' ), value: 'thursday' },
+	{ label: __( 'Friday', 'woocommerce-payments' ), value: 'friday' },
+];
+
+const monthlyAnchors = [
+	{ label: __( '1st', 'woocommerce-payments' ), value: 1 },
```

Comment:
This might be an over-optimisation (and late in the game), but we may be able to make this more succinct by adding the localized ordinal suffix on render using moment.js. E.g:

https://github.com/Automattic/woocommerce-payments/blob/93a28552fb83d225ffb6830f969bf1561bb18951/client/deposits/utils/index.js#L81-L84

Reply:
I'm leaving this alternative here but I **don't** think we should change it.

👎  It's not as easy to read
👍 there is less to translate
👍 day suffixed should be automatically localised (not tested)
❓ I feel like there must be a WP function that exists for this already

```js
function getMonthlyAnchors() {
	// Create an array of options for the monthly anchor select control.
	const monthlyAnchors = _.range( 28 ).map( ( val ) => ( {
		value: val + 1,
		label: moment()
			.date( val + 1 )
			.format( 'Do' ),
	} ) );
	// Add the last option for the last day of the month.
	monthlyAnchors.push( {
		value: 31,
		label: __( 'Last day of the month', 'woocommerce-payments' ),
	} );

	return monthlyAnchors;
}
```

Instead of:

https://github.com/Automattic/woocommerce-payments/blob/93a28552fb83d225ffb6830f969bf1561bb18951/client/settings/deposits/index.js#L38-L71